basket/news/backends/sfdc.py

```diff
@@ -345,7 +371,8 @@ def add(self, data):
         """
         if not data.get('last_name', '').strip():
             data['last_name'] = LAST_NAME_DEFAULT_VALUE
-        self.contact.create(to_vendor(data))
+
+        return self.contact.create(to_vendor(data))
```

Comment:
ah, this was the line you were mentioning earlier that returns the id, right? nice.

Reply:
yup. simple as that 🤦‍♂